express/src/lib/validators/uris-validator.ts

```diff
@@ -1,23 +1,32 @@
 import {ValidationResult} from "../../types/validation-result";
 
-export default function validateUris(urls: string[]): ValidationResult {
+export default function validateUris(urls: string[], urlLimit?: number, isGovUrl?: boolean, allowEmptyValue?: boolean): ValidationResult {
```

Comment:
I think, given we no longer use the multi URI validation, we should simplify this function a little bit and remove some of the multi URI validation stuff

Reply:
fixed